vote/vote.py

```diff
@@ -58,6 +60,31 @@ def get_votes(self):
 
         return votes
 
+    def get_least_voted(self, items, num=2):
+        """ Select (num) number of the least-voted instances
+
+        >>> v = MemeVote(test_mode=True)
+        >>> v.vote('alpha', 'beta', 'alpha')
+        >>> v.vote('beta', 'gamma', 'gamma')
+        >>> v.vote('gamma', 'delta', 'gamma')
+        >>> v.vote('alpha', 'gamma', 'alpha')
+        >>> v.get_least_voted(['alpha', 'beta', 'gamma', 'delta', 'omega'])
+        ['omega', 'delta']
+        """
+        searches = [vote['meme1'] + '|' + vote['meme2'] for vote in self.get_votes()]
+        tally = {}
+        for item in items:
+            tally[item] = len([x for x in searches if item in x])
```

Comment:
Looks like `tally` is `meme_to_comparison_count`? This took me a few min to understand. This abuses the `in` operator since it just finds substrings and there's not a guarantee that meme names will not be substrings of each other.

I think it is more clear to count and filter.

```py
meme_to_comparison_count = defaultdict(int)
for vote_dict in self.get_votes():
    meme_to_comparison_count[vote_dict['meme1']] += 1
    meme_to_comparison_count[vote_dict['meme2']] += 1
meme_to_comparison_count = {meme: count for meme, count in meme_to_comparison_count.items() if meme in items}
```